main.py

```diff
@@ -684,10 +684,9 @@ def handle_flow_mod_error(self, event):
         command = event.content.get('error_command')
         if command != 'add':
             return
-        evc_id = f'{flow.cookie:x}'
-        evc = self.circuits.get(evc_id)
+        evc = self.circuits.get(EVC.get_id_from_cookie(flow.cookie))
         if evc:
-            evc.remove_current_flows()
+            evc.remove()
```

Comment:
@ajoaoff @italovalcy once an `kytos/flow_manager.flow.error`, typically they're unrecoverable errors like `OFPT_ERROR`, except for the `SocketError/OSError` that could be temporary, assuming though that in the future we might have some enhancements on `flow_manager` to also retry for this event and abstract the retries away, further categorizing this as an unrecoverable error from the client/consumer perspective, then I figured I'd use the `remove()` to avoid operators to try to reploy this EVC to make that clearer. Let me know if you we should do this differently or if that's OK.

Reply:
Putting myself in an operator's shoes looks like having a reason why a circuit was removed would be useful, something like `disabled_reason: "some meaningful reason why"`, that way we also have it for the record in our DB and not only in the logs, wdyt?

Reply:
Good catch, @viniarck. When we introduced the cookie prefix, we forgot to update this method.